src/components/aeBanner/aeBanner.js

```diff
@@ -0,0 +1,67 @@
+const defaultOptions = {
+  timeout: 2500,
+  autoDestroy: true
+}
+
+export default {
+  name: 'ae-banner',
+  data: function () {
+    return {
+      show: false
+    }
+  },
+  props: {
+    'options': {
+      type: Object
+    }
+  },
+  mounted () {
+  },
+  methods: {
+    /**
+     * Show the banner
+     *
+     * @public
+     */
+    showBanner () {
+      if (this.show) {
+        return
+      }
+      this.show = true
+
+      if (this.allOptions.autoDestroy) {
+        this._startLazyDestroy()
+      }
+    },
+    /**
+     * Hide the banner
+     *
+     * @public
+     */
+    hideBanner () {
+      if (!this.show) {
+        return
+      }
+      this._clearTimer()
+      this.show = false
+    },
```

Comment:
Is this component visible or not should be controlled outside of this component. This component should be a dumb as AeNotification.
[Here](https://github.com/aeternity/aepp-response/blob/5ff8b09d4f48c2126f4fef8085911df93ccdc4cf/src/store/modules/response.js#L251-L256) is an example of controlling such component from Vuex store.

Reply:
Yes is controlled via the **showBanner** and **hideBanner** methods

Reply:
I mean, this component should be stateless.